### PR TITLE
Allow post loaders for template blocks

### PR DESCRIPTION
**Pull request by defnorep**
This also sets a default `lang="html"` query for template blocks.

Potentially resolves https://github.com/vuejs/vue-loader/issues/1309

### CODE REVIEW

**Comment by daekano** [comment]
First OSS PR! Tried to follow contributing standards from existing PRs, let me know if I missed anything.

File: test/template.spec.js [hunk 1]
```diff
@@ -292,3 +292,17 @@ test('separate loader configuration for template lang and js imports', done => {
...
+test('render function post loaders', done => {
+  mockBundleAndRun({
+    entry: './test/fixtures/template-post.js',
+    vue: {
+      templatePostLoaders: [
+        path.resolve(__dirname, './mock-loaders/js')
+      ]
+    }
+  }, ({ exports }) => {
+    expect(exports.staticRenderFns[0].toString()).toContain('Changed!')
```
Comment (daekano) [inline test/template.spec.js]: I'm _guessing_ this is an appropriate test. Are the static render functions invoked by the `render` function?

**Comment by daekano** [comment]
@yyx990803 Hate to bug you directly, but have you put any thought towards this PR and #1309 ?

**Comment by yyx990803** [comment]
Still thinking whether there's a better way about this. Will probably give it some proper thoughts after done with CLI.

**Comment by daekano** [comment]
Sounds good. I'll maintain the PR.

**Pull request closed without merging**